packages/app-store/paypal/lib/Paypal.ts

```diff
@@ -128,6 +128,7 @@ class Paypal {
       if (captureResult.ok) {
         const result = await captureResult.json();
         if (result.body.status === "COMPLETED") {
+        if (result?.status === "COMPLETED") {
```

Comment:
Is this expected? @alannnc

Reply:
Yes, somehow new fetcher doesn't retrieve body this time.